moveit_ros/moveit_servo/src/servo_parameters.cpp

```diff
@@ -87,7 +87,7 @@ void declareOrGetParam(T& output_value, const std::string& param_name, const rcl
  * @return true if all parameters were loaded and verified successfully, false otherwise
  */
 bool readParameters(ServoParametersPtr& parameters, const rclcpp::Node::SharedPtr& node, const rclcpp::Logger& logger,
-                    std::string ns = "moveit_servo")
+                    std::string ns /* = "moveit_servo"*/)
```

Comment:
no default? Then I would remove this comment

Reply:
I think it is nice to have the default args as comments in the code when they are now specified in a header file.  If you have a function that's defined in a header with default args you can't also specify them in the source file so this is the tactic I've used to show when looking at the code that the parameter has a default value.